docs/MergingPrs.md

```diff
@@ -15,16 +15,18 @@ Merging to `master` or `release` branches requires approval from both _main revi
 ### How to merge
 
 1. Make sure that GitHub gives the green light for merging.
-  There are a few scenarios where GitHub can prevent merging from proceeding:
-  * **Merge conflict**: The PR is conflicting with the current `master` branch; the author will
-    need to resolve the conflicts before proceeding. 
-  * **Check failure**: E.g. CI is failing.
-  * **Outdated branch**: The PR is not in sync with the current `master` branch; the author will
-    need to sync it before proceeding. This can be done via GitHub with the "Update branch" button.
+  There are a few scenarios where GitHub can prevent merging. In those cases, the dev should resolve the
+  problem as given below and submit a new iteration.
+  1. **Merge conflict**: The PR is conflicting with the current `master` branch. The author should rebase the branch
+    and resolve conflicts.
+  1. **Check failure**: E.g. CI is failing. The author should fix the causes of the check failures.
```

Comment:
Should a distinction be made between required checks and the others?

For example, this point could be made clearer if it says:

> **Required check failure**: The PR is required to pass some checks (most notably the CI tests) before it can be merged. If such checks are failing, the PR author should fix the issues causing these checks to fail.